docs/grpc_api.md

```diff
@@ -70,3 +71,28 @@ python ts_scripts/torchserve_grpc_client.py infer densenet161 examples/image_cla
 ```bash
 python ts_scripts/torchserve_grpc_client.py unregister densenet161
 ```
+## GRPC Server Side Streaming
+TorchServe GRPC APIs adds a server side streaming of the inference API "StreamPredictions" to allow a sequence of inference responses to be sent over the same GRPC stream. This new API is only recommended for the use case when the inference full response latency is high, and the inference intermediate results are sent to client. This new API automatically forces the batchSize to be one.
```

Comment:
maybe we can add one line here, 
`This new API is only recommended for use cases such as generative AI and mostly LLMs (Large Language models) where the latency for the full response is high, and the intermediate results are available and can be sent to client as they get generated. This new API automatically forces the batchSize to be one.
`